docs/_docs/14-helpers.md

```diff
@@ -240,6 +241,32 @@ header:
     provider: google-drive
 ```
 
+### bilibili
```

Comment:
Typesetting consistency: A heading should always begin with an uppercase letter.

Also an empty line below the heading line would also make it more consistent with other headings in the document.